src/ui/component/DisciteModule.html

```diff
@@ -8,8 +8,10 @@ <h3 class="heading bold-small heading--text">
             </a>
             <div class="discite__area {{modifier ? `discite__area--${modifier}` : ''}}">
                 <p class="font-small discite__counter">
-                    This is a module in <a href="/courses/{{module.course.id}}"><b>{{ module.course.title }}.</b></a> There are {{module.course.modules.length}} in total.
-                </p>
+					This is a module in <a href="/courses/{{module.course.id}}"><b>{{ module.course.title }}.</b></a>
+				</p>
+				<p></p>
```

Comment:
What's this empty `<p>` for? If for spacing please add some margin or padding to the `.discite__counter` class instead